pkg/kubefed2/federate/type.go

```diff
@@ -59,17 +59,17 @@ var (
 		the federation control plane. Please use the
 		--host-cluster-context flag otherwise.`
 
-	enable_example = `
-		# Enable federation of Services with service type overrideable
-		kubefed2 federate enable Service --override-paths=spec.type --host-cluster-context=cluster1`
+	type_example = `
+		# Federate type Services with value of field SessionAffinity overrideable
```

Comment:
Now that generic overrides have merged mention of override-paths can be removed.